ocean_lib/ocean/ocean_assets.py

```diff
@@ -214,15 +215,15 @@ def create(
             dt = DataToken(data_token_address)
             minter = dt.contract_concise.minter()
             if not minter:
-                raise AssertionError(
+                raise InvalidDatatokenContract(
                     f"datatoken address {data_token_address} does not seem to be a valid DataToken contract."
                 )
             elif minter.lower() != publisher_wallet.address.lower():
-                raise AssertionError(
+                raise InvalidDatatokenMinter(
```

Comment:
e.g.  web3 ValidationError or some Python generic error?

Reply:
I just checked Python generic errors at https://www.tutorialsteacher.com/python/error-types-in-python

None of them are appropriate.

Web3.ValidationError is sufficient.